include/functions.h

```diff
@@ -2529,7 +2529,7 @@ s32 __osCheckPackId(OSPfs* pfs, __OSPackId* temp);
 s32 __osGetId(OSPfs* pfs);
 s32 __osCheckId(OSPfs* pfs);
 s32 __osPfsRWInode(OSPfs* pfs, __OSInode* inode, u8 flag, u8 bank);
-// ? func_80102FA0(?);
+// ? guMtxL2F(?);
```

Comment:
can probably add the prototype here since its used externally.

also, zap2 changes? did we downgrade on accident at some point or something

Reply:
#452 downgraded ZAP2 it looks like?

Reply:
rip lmfao, thanks